sysutils/pagerduty-notifier/Portfile

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/trustpilot/pagerduty-notifier 20200817-0652
+
+description         PagerDuty native macOS notifications
+
+long_description    PagerDuty Notifier polls the PagerDuty API for new \
+                    incidents and sends native macOS notifications. You can \
+                    control the notification style as you normally would in \
+                    System Preferences.
+
+categories          sysutils
+license             MIT
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+installs_libs       no
+
+build.cmd           make
+build.target        build
+build.env-append    GO111MODULE=off \
+                    GOPROXY=off
```

Comment:
These lines won't be necessary pending #8811.

Reply:
Not an issue, I'll remove them afterwards, or on the next update.